### PR TITLE
Add my card information

### PR DESCRIPTION
**Pull request by KyrtovaLera**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://www.google.com.ua/" target="_blank" title="Third resource">
+                  Google
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (dilshankarunarathne) [inline index.html:146]: Please change this comment too, by adding your name here.

File: index.html [hunk 2]
```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
...
         <!-- ________ Shiva's card START ________  -->
```
Comment (dilshankarunarathne) [inline index.html:147]: Blank lines are important when others read this code afterwards.

File: index.html [hunk 3]
```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
```
Comment (dilshankarunarathne) [inline index.html:116]: Please leave blank lines around your card as per the instructions (one line before - and one line after).

**Review by dilshankarunarathne — CHANGES_REQUESTED** [review]
Hello @KyrtovaLera, once you make these changes - please get back to us, we will be able to review and merge your pull request.

**Comment by Preet-Sojitra** [comment]
Congrtulations @KyrtovaLera  your pr has been merged. Have a nice day ahead :)

**Pull request merged**